demo/grpc-api/src/main/docker/Dockerfile

```diff
@@ -4,4 +4,9 @@ FROM deephaven/grpc-api:local-build
 COPY libs/ /app/libs
 
 # Add the application directory
-COPY app.d/ /app.d
\ No newline at end of file
+COPY app.d/ /app.d
+
+
+from python:3.8
+COPY requirements.txt /requirements.txt
+RUN pip install -r /requirements.txt
```

Comment:
This isn't going to work - this is just creating a new image with python instead of basing off of deephaven/grpc-api.

Reply:
I thought I removed line 10 and 11 and changed it to :RUN pip3 install -r /app.d/requirements.txt

Reply:
This is also misleading, the branch I'm deploying no longer uses this Dockerfile (though, arguably, it should again; I think it was removed when I was away and mike 'n devin were trying to get a working deploy...). My repo+branch is JamesXNelson/deephaven-core jxn/demo_refactored and in it, the base `docker/server/src/main/docker/Dockerfile` is what is getting deployed.  This is very messy, and needs to be cleaned up....  but, for now, we need to give _that_ image the correctly installed py dependencies, preferably using pre-baked base images that already have all python dependencies setup correctly.

Reply:
note, I select the image in demo/demo.gradle via
`t.dependsOn demoGrpcApi.tasks.findByName('buildDocker-server')`

If we setup a "has everything" base image in `docker/server`, then I can just point to it via task name in demo.gradle, and we should be done here.